micro_brewery/demo/sale_order.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="sale_order_1" model="sale.order">
+        <field name="origin">The Wobbly Walrus Pub's opportunity</field>
+        <field name="opportunity_id" ref="crm_lead_8"/>
+        <field name="partner_id" ref="res_partner_17"/>
+    </record>
+    <record id="sale_order_3" model="sale.order">
+        <field name="origin">The Wobbly Walrus Pub's opportunity</field>
+        <field name="opportunity_id" ref="crm_lead_8"/>
+        <field name="partner_id" ref="res_partner_15"/>
+    </record>
+    <record id="sale_order_4" model="sale.order">
+        <field name="origin">The Drunken Donkey Inn's opportunity</field>
+        <field name="opportunity_id" ref="crm_lead_4"/>
+        <field name="partner_id" ref="res_partner_13"/>
+        <field name="sale_order_template_id" ref="sale_order_template_1"/>
+    </record>
```

Comment:
change the sequence 

Assume you are creating real sale order and what should you need to add first?